Example/SettingsViewController.swift

```diff
@@ -95,8 +95,8 @@ class SettingsViewController: UITableViewController {
         if let selectedOfflineVersion = Settings.selectedOfflineVersion {
             let tilesURL = Bundle.mapboxCoreNavigation.suggestedTileURL(version: selectedOfflineVersion)
             
-            Settings.directions.configureRouter(tilesURL: tilesURL!) { [weak self] (numberOfTiles) in
-                let message = String.localizedStringWithFormat(NSLocalizedString("ROUTER_CONFIGURED_MSG", value: "Router configured with %ld tile(s).", comment: "Alert message when a router has been configured; 1 = number of map tiles"), numberOfTiles)
+            Settings.directions.configureRouter(tilesURL: tilesURL!) { [weak self] in
+                let message = String.localizedStringWithFormat(NSLocalizedString("ROUTER_CONFIGURED_MSG", value: "Router was configured.", comment: "Alert message when a router has been configured."))
```

Comment:
This backwards compatibility issue in MapboxNavigationNative was rolled back in v14.1.4, which I think we can upgrade to pretty smoothly from v14.1.3. I guess we might as well bring back the callback argument.

Reply:
Done.